content/docs/gke/deploy/minikf-gcp.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Deploy using MiniKF on GCP"
+description = "Instructions for using the Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)."
+weight = 4
++++
+
+This guide describes how to launch a MiniKF VM on GCP. MiniKF is a
+production-ready, full-fledged Kubeflow deployment that installs in minutes.
+It's a fast and easy way to get started with Kubeflow. With just a few clicks,
+you are up for experimentation, and for running complete Kubeflow Pipelines.
+
+MiniKF installs Kubeflow 0.7.0 and supports NVIDIA GPUs.
+
+### Installing MiniKF on GCP
+
+To install MiniKF on GCP, follow the steps below:
+
+Go to the [MiniKF page](To install MiniKF on GCP, follow the steps below) on
+Google Cloud Marketplace.
+
+Click on the Launch on Compute Engine button
+
+<p class="text-center">
```

Comment:
For image styling, use this code pattern:

```
    <img src="/docs/images/my-image.png" 
      alt="OAuth consent screen"
      class="mt-3 mb-3 p-3 border border-info rounded">
```

You can see an example in this file: 
https://raw.githubusercontent.com/kubeflow/website/master/content/docs/gke/deploy/oauth-setup.md

Reply:
thanks!